### ISSUE TITLE
tramp filenames for predefined breakpoints

### ISSUE THREAD

**Issue by ksjogo** [ahungry/geben#42]
Should/could we strip the tramp prefix before adding the breakpoint to the predefined list.

**Comment by mallt** [comment]
Hi @ksjogo, do you mean you want to debug remote files opened with tramp? I'm always debugging on my local machine so I'm not really familiar with this setup.

**Comment by ksjogo** [comment]
Yeah, exactly. So tramp has some remote file opened. add-predefined-breakpoint will keep the /ssh:hostythings:/file, but xdebug will not accept that. I suppose we could just strip the ssh prefix and host
